step-definitions/core/core.js

```diff
@@ -3,7 +3,7 @@ import { Given, When, Then } from 'cucumber';
 
 import { setupEvan } from '../../test-utils/test-utils.js';
 
-export const WAIT_TIME = 15 * 1000;
+export const WAIT_TIME = 300 * 1000;
```

Comment:
I'm unsure if it's a good idea to "globally" increase the wait time at such scale, while we only really expect such wait times during eve payments?